image/templates/helm/stackrox-secured-cluster/templates/_defaults.tpl

```diff
@@ -33,3 +33,26 @@
   {{ end }}
 {{ end }}
 {{ end }}
+
+{{/*
+  srox.ensureCentralEndpointContainsPort .
+
+  Appends a default port to the configured central endpoint based on a very simply heuristic.
+  Specifically, it only checks if the provided endpoint contains a prefix "https://" and
+  the part after that prefix does not contain a double colon.
+  This heuristic is kept simple on purpose and does not correctly add the default ports in
+  case the host part is an IPv6 address.
+*/}}
+{{ define "srox.ensureCentralEndpointContainsPort" }}
+  {{ $ := . }}
+
+  {{ $endpoint := $._rox.centralEndpoint }}
+  {{ if hasPrefix "https://" $endpoint }}
+    {{ $endpoint = trimPrefix "https://" $endpoint }}
+    {{ if not (contains ":" $endpoint) }}
+      {{ include "srox.warn" (list $ (printf "Specified centralEndpoint %s is missing a port, assuming the port is 443. If this is wrong please specify the correct port." $._rox.centralEndpoint)) }}
```

Comment:
Is it really a warning? It looks like a correct setup to me. Maybe change it to info?

Reply:
I think you are right, I have converted it to a note. Thanks.